css/pages/signup.css

```
@@ -0,0 +1,3 @@
+body {
+    margin: 60px 640px;
```

❗️ 수정요청
해당 속성으로 인해 div.input-info 태그가 400px 만큼 커질 수 있을 때도 충분한 영역을 가지지 못하고 있는 것으로 보입니다~
아래처럼 작성해주시는 것을 추천드려요!

```suggestion
    margin: 60px auto;
```